src/styles/_card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  height: 408px;
+  background-color: white;
+  &__photo {
+    width: 160px;
+    height: 134px;
+    padding: 33px 20px 0;
+  }
+  &__title {
+    color: var(--main-color);
+    text-decoration: none;
+    display: block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    padding: 36px 17px 0;
+    margin: 0;
+  }
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    color: var(--second-color);
+    padding: 4px 0 0 17px;
+  }
+  &__reviwes {
+    padding: 17px 17px 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .reviews-count {
+      font-weight: 400;
+      font-size: 10px;
+      color: var(--main-color);
+    }
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 23px 17px 0;
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      color: var(--second-color);
+    }
+    &-number {
+      color: var(--main-color);
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+  &__button {
+    margin-left: 17px;
+    margin-top: 16px;
+    background-color: #00acdc;
+    width: 164px;
+    height: 40px;
```

Comment:
the same with hardcoding height for elements with content inside. Use padding instead